### ISSUE TITLE
Command should continue if one package fails.

### ISSUE THREAD

**Issue by ShoeUnited** [Kiwi/clyde#22]
With clyde -Rd foobar1 foobar2  if you don't have foobar1 it prints an error and all that which is normal and cool.  The problem though is that it quits. >.>   If I try to -Rd multiple items and one of them is not actually spelled right or installed it won't touch the others in the line.

So my question was that even though it can't find foobar1 it could still proceed to remove the next thing(s) in the list?  I think this could be handy if I say mistyped foobar1 as foobars1, that way the error is produced but it still takes care of the rest.

-Shoe

**Comment by nogweii** [comment]
This is true of other commands as well; -S I know suffers. e.g. `clyde -S fake-pkg real-pkg` fails on fake-pkg but doesn't even check for real-pkg.
